server.js

```diff
@@ -26,6 +16,12 @@ app.use(require("webpack-dev-middleware")(compiler, {
 }));
 app.use(require("webpack-hot-middleware")(compiler));
 
+//  Create hot-reloading bundle for server rendering
+var basePath = require('path').resolve(__dirname, '.');
+var webpackIsomorphicTools = new WebpackIsomorphicTools(require('./webpack-isomorphic-tools'))
+    .development(true)
+    .server(basePath, function() { });
```

Comment:
Yeah it feels quite magic.

As I understand it takes the assets generated by webpack and maps them correctly when requiring them. 

Do you maybe know a better way to do this? I could imagine even reusing the existing `compiler`.